### PR TITLE
add airflow clickhouse plugin

### PR DESCRIPTION
**Pull request by noobmdev**

### CODE REVIEW

File: docker/airflow-clickhouse/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Ref: https://airflow.apache.org/docs/docker-stack/build.html#example-of-adding-pypi-package
+
+FROM apache/airflow:2.6.3
+USER airflow
+RUN pip install --no-cache-dir "apache-airflow==${AIRFLOW_VERSION}" airflow-clickhouse-plugin==0.11.0
```
Comment (trmaphi) [inline docker/airflow-clickhouse/Dockerfile:5]: why use parameterized airflow version when you already use tag version in base image

File: dags/example_clickhouse_operator.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+from airflow import DAG
+from airflow_clickhouse_plugin.operators.clickhouse_operator import ClickHouseOperator
+from airflow.operators.python_operator import PythonOperator
+from datetime import datetime, timedelta
+
+default_args = {
+    'owner': 'airflow',
+    'depends_on_past': True,
+    'start_date': datetime(2023, 8, 4),
```
Comment (trmaphi) [inline dags/example_clickhouse_operator.py:9]: do we need start_date for example?

**Review by trmaphi — CHANGES_REQUESTED** [review]

File: dags/example_clickhouse_operator.py (hunk 2 above)
Comment (noobmdev) [reply dags/example_clickhouse_operator.py:9]: > do we need start_date for example?

start_date is required field of DAG

File: docker/airflow-clickhouse/Dockerfile (hunk 1 above)
Comment (noobmdev) [reply docker/airflow-clickhouse/Dockerfile]: > why use parameterized airflow version when you already use tag version in base image

Updated to default AIRFLOW_VERSION argument

**Pull request merged**